src/pytorch_lightning/core/optimizer.py

```diff
@@ -53,6 +53,7 @@ def __init__(self, optimizer: Optimizer):
     def optimizer(self) -> Optimizer:
         return self._optimizer
 
+    # TODO:
     @classmethod
     def _to_lightning_optimizer(
         cls, optimizer: Union[Optimizer, "LightningOptimizer"], strategy: "pl.strategies.Strategy", opt_idx: int
```

Comment:
Here is also an opt_idx argument. But this is only used internally for bookkeeping (we loop over optimizers and enumerate them to call this method). It would be better to remove this in a follow up, separate from this PR.